api/agent/agent_test.go

```diff
@@ -412,6 +422,7 @@ func TestSubmitError(t *testing.T) {
 		"FN_APP_NAME": appName,
 		"FN_PATH":     path,
 		"FN_MEMORY":   strconv.Itoa(memory),
+		"FN_CPUS":     CPUs,
```

Comment:
Overall looks good, but i do feel like there would be a problem with UX. How would user know the capacity for CPU and RAM? I do feel like there should be some sort of API to see the exact free/used capacity (think of flavors on AWS, OpenStack, etc.).

Reply:
Yes, I need to add these as stats (CPU, mem, slots in use, hot/cold container counts, etc. etc.) and maybe a API could also be useful. I'll open an issue to discuss and proceed.